src/browser/HostInstaller.cpp

```diff
@@ -38,11 +38,13 @@ HostInstaller::HostInstaller()
     , TARGET_DIR_CHROMIUM("/Library/Application Support/Chromium/NativeMessagingHosts")
     , TARGET_DIR_FIREFOX("/Library/Application Support/Mozilla/NativeMessagingHosts")
     , TARGET_DIR_VIVALDI("/Library/Application Support/Vivaldi/NativeMessagingHosts")
+    , TARGET_DIR_TOR_BROWSER("/Library/Application Support/TorBrowser-Data/Browser/Mozilla/NativeMessagingHosts")
 #elif defined(Q_OS_LINUX)
     , TARGET_DIR_CHROME("/.config/google-chrome/NativeMessagingHosts")
-    , TARGET_DIR_CHROMIUM("/.config/chromium/NativeMessagingHosts")
+    , TARGET_DIR_CHROMIUM("/.config/chromium/NativeMessagingHosts")	
```

Comment:
You should remove the spaces a tthe end of the line.

Reply:
Vim guy - apologies.